integration/tests/access/cohort3/grpc_streaming_blocks_test.go

```diff
@@ -187,8 +182,12 @@ func (s *GrpcBlocksStreamSuite) TestHappyPath() {
 					foundANTxCount++
 				case block := <-observerBlocks:
 					s.T().Logf("ON block received: height: %d", block.Header.Height)
-					r.Add(s.T(), block.Header.Height, "observer", block)
-					foundONTxCount++
+
+					// check only that responses for ON which was tracked by AN and compare them
+					if len(r.r[block.Header.Height]) > 0 {
```

Comment:
let's only gate this for tests subscribing from the latest block. the other tests should always have matching heights

Reply:
[4b0110c](https://github.com/onflow/flow-go/pull/6125/commits/4b0110c3da273b88d37a97b90e081c1d0b554714)